### ISSUE TITLE
Windows does not have / display properly the fonts or characters.

### ISSUE THREAD

**Issue by ctunited15** [sonatype-nexus-community/cargo-pants#53]
I ran the command _cargo pants_ from a Git bash window on my Windows machine.  It completed successfully, but the output display couldn't find the proper characters.
![cargo_pants_windows](https://user-images.githubusercontent.com/60018862/126876956-b2edcae4-8cd5-4cb5-b9fe-26c5cf54c752.png)

**Comment by amy-keibler** [comment]
I could be misreading your screenshot, but it appears that you have `cargo-pants` version `0.1.26` installed. We've recently published [version `0.3.1`](https://crates.io/crates/cargo-pants/0.3.1), so upgrading might help with this issue. We recently switched how we display tables and that might solve this problem.

If upgrading doesn't solve the issue, we're using [term-table's `rounded`](https://docs.rs/term-table/1.3.1/term_table/struct.TableStyle.html#method.rounded) styling to display the table and we could add a command-line flag to switch that to [`simple`](https://docs.rs/term-table/1.3.1/term_table/struct.TableStyle.html#method.simple), which is more likely to display correctly. I do not have a windows machine to test this, so I likely cannot reproduce or test the issue. My guess is that we're outputting text as UTF-8, but the terminal expects a different encoding. I found [a stackoverflow page](https://stackoverflow.com/questions/10651975/unicode-utf-8-with-git-bash) of things I'd try if I could reproduce the issue, so maybe that might point you in the right direction until we have time to address this on our end?

I am also interested in the `lock&#95;` api crate part of the line starting with "Vulnerability Title:". I can reproduce that issue, so I will make a separate ticket to address that.

Thank you for filing this issue so we can ensure a good experience for users on other platforms and configurations.
